.github/workflows/cypress-workflow.yml

```diff
@@ -7,8 +7,8 @@ on:
     branches:
       - "*"
 env:
-  OPENSEARCH_DASHBOARDS_VERSION: '2.6'
-  OPENSEARCH_VERSION: '2.6.0-SNAPSHOT'
+  OPENSEARCH_DASHBOARDS_VERSION: '2.7'
```

Comment:
Change to 2.x
And on line 32: `ref: ${{ env.OPENSEARCH_DASHBOARDS_VERSION }}`

Reply:
Updated. Thanks @bowenlan-amzn